### ISSUE TITLE
[Content] My VA: Post usability testing content updates

### ISSUE THREAD

**Issue by tressaellen** [department-of-veterans-affairs/va.gov-team#21543]
## Background

We've made all the updates from the usability testing and now we need some content changes, specifically all of the CTA's need to be re-evaluated. We took out all of the totals and we're now looking for a more generic 'Manage all X' or 'Go to all X', something along those lines. (All the ones that need to be looked at are outlined in a red box)

 Can you give use some guidance on the best practice here?

![My VA Desktop Updates Copy.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/803ae2d2-07c2-4315-add4-b04a4f8e5b26)

## Tasks

- [ ] Peggy give copy guidance
- [ ] Implement all changes and relay to FE

**Comment by bethpottsVADEPO** [comment]
@tressaellen Can you give me the test user info for the above screen shot?

**Comment by bethpottsVADEPO** [comment]
@tressaellen Also, can you tell me what the labels were that didn't test well?

**Comment by tressaellen** [comment]
Hi @bethpottsVADEPO - we haven't tested this particular design yet, we're going to do another round of testing after launch with users real information. (Is that what you were asking?)

And yes - what we tested with is below.

![My VA desktop 1](https://user-images.githubusercontent.com/58752772/111332150-636e8a00-8626-11eb-802e-4a02e337e00d.jpg)

**Comment by bethpottsVADEPO** [comment]
@tressaellen thanks! I found your research synth in the design channel, so I got the full picture. I agree with your approach. And I'd love to have "your" in there if you think there's space. I do see "Manage all your prescriptions" but then these 2 don't have "your":
Manage all claims and appeals --> Manage all your claims and appeals
Manage all appointments -->  Manage all your appointments

cc @peggygannon

**Comment by tressaellen** [comment]
Hi @bethpottsVADEPO - thanks so much for the feedback! I added 'your' to Just so you're aware, there's a been a bit of a change to the overall design. I've updated the links to reflect what is shown on the Pittsburgh site, let me know if you have any additional changes on those. 

The only link that needs to be looked at is if we can't reach the endpoint for messages (shown in the second image), we are going to downgrade it to a generic message. I also copied this link from the Pittsburgh site but wanted to get your thoughts as well. 

![My VA desktop 2.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/bd13cddc-6af6-4b1f-b234-8fcc3589d7ac)

![My VA Desktop 2 no messages.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/98482dc0-f01b-43b2-be84-1fa49036eaf1)
